src/main/scala/lisa/settheory/SetTheoryZAxioms.scala

```diff
@@ -19,7 +19,7 @@ private[settheory] trait SetTheoryZAxioms extends SetTheoryDefinitions {
    *
    * `() |- (x = y) ⇔ ∀ z. z ∈ x ⇔ z ∈ y`
    */
-  final val extensionalityAxiom: this.AXIOM = Axiom("extensionalityAxiom", forall(z, in(z, x) <=> in(z, y)) <=> (x === y))
+  final val extensionalityAxiom: this.AXIOM = Axiom(forall(z, in(z, x) <=> in(z, y)) <=> (x === y))
```

Comment:
```suggestion
  final val extensionalityAxiom: AXIOM = Axiom(forall(z, in(z, x) <=> in(z, y)) <=> (x === y))
```

The type annotations in this file are slightly inconsistent, with just this one being different afaict